package.json

```diff
@@ -30,6 +30,7 @@
     "jsdom": "^9.2.1",
     "mocha": "^2.5.3",
     "react": "^15.1.0",
+    "react-addons-test-utils": "^15.1.0",
```

Comment:
Add react test utils to simulate events. We have some issues using `dispatchEvent` with detached element in the test: https://github.com/starsirius/crystals/pull/5/files#diff-a7f596f23edd58b2bbfe338299304936R161